src/main/kotlin/commands/BanCommand.kt

```diff
@@ -0,0 +1,72 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import ConfigType
+import Main
+import Permissions.hasPermission
+import Send.error
+import UserConfig
+import arg
+import doesLater
+import greedyString
+import integer
+import net.ayataka.kordis.entity.user.User
+import net.ayataka.kordis.exception.MissingPermissionsException
+import string
+
+object BanCommand : Command("ban") {
+    init {
+        string("user") {
+            integer("deleteMessageDays") {
+                greedyString("reason") {
+                    doesLater { context ->
+                        if (!message.hasPermission(PermissionTypes.COUNCIL_MEMBER)) {
+                            return@doesLater
+                        }
+                        val username: String = context arg "user"
+                        val serverId = readConfigSafe<UserConfig>(ConfigType.USER, false)?.primaryServerId
+                        val user: User = Main.client?.servers?.find(serverId ?: 573954110454366214)?.members?.findByName(username) ?: //username
+                        Main.client?.servers?.find(serverId ?: 573954110454366214)?.members?.find { it.nickname == username } ?: //nick
+                        Main.client?.servers?.find(serverId ?: 573954110454366214)?.members?.find(username.toLong()) ?: //id
+                        run {
+                            message.error("User not found!")
+                            return@doesLater
+                        }
+                        val messageDays: Int = context arg "deleteMessageDays"
```

Comment:
There's really no reason for this. By default we shouldn't delete any messages, and otherwise it should only be a day, if someone was breaking the rules for an entire week you have a bigger issue on hand